libraries/AP_AccelCal/AP_AccelCal.cpp

```diff
@@ -56,7 +56,10 @@ void AP_AccelCal::update()
             case ACCEL_CAL_WAITING_FOR_ORIENTATION: {
                 // if we're waiting for orientation, first ensure that all calibrators are on the same step
                 uint8_t step;
-                cal = get_calibrator(0);
+                if ((cal = get_calibrator(0)) == NULL) {
```

Comment:
Yeah, fair enough, I'll fix that.  I actually did it the normal way first.. and doing it this way made me think.. hmm.. is this ok?